components/docs/PluginsList.tsx

```diff
@@ -5,7 +5,11 @@ import { PluginManifestItem } from "../../scripts/generate_plugin_json";
 
 const PluginDocsLink = (plugin: PluginManifestItem) => {
   if (!plugin.docsUrl) return <span>{plugin.name}</span>;
-  return <Link href={plugin.docsUrl}>{plugin.name}</Link>;
+  return (
+    <Link href={plugin.docsUrl.replace("https://www.grouparoo.com", "")}>
+      {plugin.name}
+    </Link>
+  );
```

Comment:
Make these links relative, so they work on localhost too

Reply:
Could we make them relative at generation, or would that cause other issues?

Reply:
I think we want the FQDN in the data file, as that's what builds our app manifest that core uses